packages/vue/src/index.ts

```diff
@@ -163,6 +184,28 @@ const Frontegg: PluginObject<PluginOptions> | any = {
       }
     }
 
+    if (isVue3) {
+      // @ts-ignore - provide will exist only in vue 3 app
```

Comment:
cant we give it vue3 types?

Reply:
unfortunely no as Vue is typed as Vue 2 app